ush/global_cycle.sh

```diff
@@ -393,8 +395,9 @@ EOF
 cat << EOF > fort.37
  &NAMSFCD
   NST_FILE="$NST_FILE",
```

Comment:
JEDI_SNO_FILE and JEDI_SOI_FILE should be set the same way. The default should be NULL and they should be added to the fort.37 namelist.

Reply:
For now, only GSI_SOI_FILE is set in the fort.37 namelist. JEDI_SNO_FILE name is fixed within READ_DATA (i.e., xainc.00$NN). Just to be consistent with JEDI SNOW update, I also fixed the JEDI_SOI_FILE name in the READ_DATA as soil_xainc.00$NN. I am trying to be consistent with the current code norm but am happy to adjust as well.

Reply:
If you want to fix the name of JEDI_SOI_FILE in the source code, then I would update `global_cycle_driver.sh` as follows: 

```
   if [[ "$DO_SOI_INC_JEDI" == ".true." ]] ; then
-        ln -fs $HOMEgfs/tests/global_cycle/data/soil_xainc.00$n $DATA/.
+        ln -fs $HOMEgfs/tests/global_cycle/data/soil_xainc.00$n $DATA/soil_xainc.00$n
   fi
 done
```

Then, if the file name ever changes, we just have to change the script and not the source code.

Reply:
Thanks, will do - if we eventually decide to stick to the current reading mode. Once Clara gets a chance to chime in, we can figure out which option is better, i.e., namelist vs. fixed in the source code.

Reply:
I see now that in my review, I advised the opposite of George, in terms of how to make these consistent. I don't mind which way you go (all controlled by file name, or all controlled by logicals ) - they just should all be the same.

Reply:
OK. now edits were done including 1) make the three options consistent and 2) update global cycle driver to avoid editing source code for a different file name.